pontoon/administration/files.py

```diff
@@ -1229,7 +1229,7 @@ def dump_from_database(project, locale):
     # Asymmetric formats: Remove l10n files from locale repository
     asymmetric = ['dtd', 'properties', 'ini', 'inc']
 
-    if all(x in formats for x in asymmetric):
+    if all(x in asymmetric for x in formats):
```

Comment:
We've got `Resource.is_asymmetric` now, how about we just loop through the resources and use that?

Also, the PR comment mentions it should be considered asymmetric if _any_ resource is asymmetric, but you're using `all` here. Should that be `any`?